### ISSUE TITLE
Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (.btn-link)

### ISSUE THREAD

**Issue by Govardhani-T** [microsoft/vscode-docs#7369]
**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver".**
### GitHub Tags:
#A11yTCS; #A11ySev2; #A11yMAS; #DesktopWeb; #Chrome; #WCAG1.4.3; #WCP; #Contrast(minimum); #Visual Studio Code Services; #A11yAuto; #AI4W; #BM_VisualStudioCodeServices_Web_Apr2023; #Win11;#color-contrast;
### Environment Details: 
Application Name:  code.visualstudio.com
URL: https://code.visualstudio.com/download?dark-plus-v2
Microsoft Edge version 125.0.0.0 (Official Build) (64-bit)
### Repro Steps:
1. Hit the URL: https://code.visualstudio.com/download?dark-plus-v2
2. Run fast pass and observe the issue.
### Issue: 
Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (color-contrast - https://accessibilityinsights.io/info-examples/web/color-contrast)
### Target application: 
Download Visual Studio Code - Mac, Linux, Windows - https://code.visualstudio.com/download?dark-plus-v2
### Element path:
 .btn-link
### Snippet: 
<button class="btn-link sha-table-btn" style="font-size: 14px; font-weight: 150;">See SHA-256 Hashes</button>
### Related paths: 
  #theme-v2
### How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 4.23 (foreground color: #337ab7, background color: #0e0e0e, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1

Environment: Microsoft Edge version 125.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.42.0 (axe-core 4.8.4), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
###Attachment:
![image](https://github.com/microsoft/vscode-docs/assets/172380684/4ce9e78d-db5c-47bb-88f0-73c9a6d3ebf9)

**Comment by meganrogge** [comment]
@Govardhani-T, @rzhao271 says this issue now passes FastPass, could you please test to confirm?

**Comment by Govardhani-T** [comment]
Verified the issue in the environment. Issue is fixed. 
<img width="957" alt="issue fixed" src="https://github.com/user-attachments/assets/def0a15c-c3b0-4fbf-8ab9-60abd25a56a3">
